components/automate-ui/src/app/entities/layout/layout-sidebar.service.ts

```diff
@@ -35,6 +35,16 @@ export class LayoutSidebarService {
     private populateSidebar() {
       const sidebars: Sidebars = {
         active: '',
+        desktop: [{
+          name: 'Desktop',
+          items: [
+            {
+              name: 'Desktop',
+              icon: 'today',
+              route: '/desktop'
```

Comment:
We should not need this.

Reply:
Ideally? Can you please elaborate? 😃

Reply:
This was a note for me to try and move this section.

Reply:
This was a note for me to try and remove this section.